playwright.config.js

```diff
@@ -16,6 +18,8 @@ export default defineConfig({
     baseURL: 'https://www.globalsqa.com/',
     testIdAttribute: 'id'
   },
+      timeout: 60000,  // Global timeout for all tests ***
+     //  retries: 2,  // ***
```

Comment:
The `retries` setting is commented out. If you intend to use it, make sure to uncomment it. Otherwise, it will not take effect.